### PR TITLE
Add tests for edge case and modify view. all test pass.

### PR DESCRIPTION
**Pull request by jequeza**
This branch adds the functionality to display the items for a given invoice. Along with the data that links an invoice to an item, quantity, unit price, and status. All tests are passing.

### CODE REVIEW

File: app/views/invoices/show.html.erb [hunk 1]
```diff
@@ -7,4 +7,24 @@
 <section id="invoice-customer-info">
   <h2>Customer:</h2>
   <p><%= @merchant_invoice.customer.name %></p>
-</section>
\ No newline at end of file
+</section>
+
+<section id="invoice-items-info">
+  <h3>Items on this Invoice: </h3>
+  <table>
```
Comment (klatour324) [inline app/views/invoices/show.html.erb:14]: Yes! Wonderful use of a table here so we get this displayed nicely on the show page!

**Comment by klatour324** [comment]
Looks good. Great catch on adding the edge case to testing and making sure we are only seeing the items for the specific merchant.
